src/main/java/net/atos/client/bag/BAGClientService.java

```diff
@@ -37,55 +36,49 @@ public class BAGClientService {
 
     public static final String DEFAULT_CRS = "epsg:28992";
 
-    private static final String ADRES_EXPAND = "panden, adresseerbaarObject, nummeraanduiding, openbareRuimte, woonplaats";
+    private static final String ADRES_EXPAND =
```

Comment:
I guess the max line length is now the default 120 right? For Kotlin we usually set it to 140. Is that maybe possible here too? I think 120 is a little conservative. ;-)

Reply:
I think it's even 80 chars by default and not possible to change.

A lot of things are not possible with google formatter:
* having parameters offset based on where the method parenthesis starts, so they are aligned pretty
* line length
* no configuration for lots of things (by design)

We might want to switch to Eclipse formatter but that means configuring it via Eclipse (not IntelliJ).

Reply:
https://github.com/diffplug/spotless/issues/134

Reply:
Hmm, maybe you are right yes..

Reply:
We're now using 140 chars